src/core/src/package_managers/YumPackageManager.py

```diff
@@ -646,31 +653,24 @@ def backup_image_default_patch_configuration_if_not_exists(self):
                             "apply_updates": "yes/no/empty string",
                             "download_updates": "yes/no/empty string",
                             "enable_on_reboot": true/false,
-                            "install_state": true/false
+                            "installation_state": true/false
                         },
                         "dnf-automatic": {
                             "apply_updates": "yes/no/empty string",
                             "download_updates": "yes/no/empty string",
                             "enable_on_reboot": true/false,
-                            "install_state": true/false
+                            "installation_state": true/false
                         },
                         "packagekit": {
                             "WritePreparedUpdates": "true/false/empty string",
                             "GetPreparedUpdates": "true/false/empty string", //NOTE: This property name is pending validation as noted in another comment where the name is initialized
                             "enable_on_reboot": true/false,
-                            "install_state": true/false
+                            "installation_state": true/false
```

Comment:
Isn't this a breaking change for machines that have already stored information in the previous format?